helloSchnauf/scripts/prettifyChangedFiles.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+jsfiles=$(git diff --cached --name-only --diff-filter=ACM "*.js" "*.jsx" | tr '\n' ' ')
```

Comment:
Kurze Doku wäre gut. Quite the Bobby magic in here 😆

Reply:
ja. leider auch ka